test/TestVariableHasSpaces.py

```diff
@@ -31,6 +31,10 @@
 - name: bad variable format
   debug:
     msg: "{{bad_format }}"
+- name: not a jinja variable
+  debug:
+    msg "test"
```

Comment:
I guess this doesn't have to be vaild yaml but missing a ":"

Reply:
thanks, i corrected